src/SourceGenerators/SourceGeneratorHelpers/Helpers/SymbolExtensions.cs

```diff
@@ -599,20 +599,21 @@ private static string GetFullyQualifiedType(this ITypeSymbol type, bool includeG
 		/// </summary>
 		/// <param name="propertyOrSetter">The dependency-property or the attached dependency-property setter</param>
 		/// <returns>The property type</returns>
-		public static INamedTypeSymbol? FindDependencyPropertyType(this ISymbol propertyOrSetter)
+		public static INamedTypeSymbol? FindDependencyPropertyType(this ISymbol propertyOrSetter, bool unwrapNullable = true)
```

Comment:
Why would any caller need to unwrap. It doesn't make sense to me (I see it's existing code, but wondering if we should not unwrap for ALL callers)

Reply:
i would tent to agree with you, but im afraid there are cases where it matters and breaks things
ill probably remove that, in a follow up pr to better assess the impact, in a more isolated context